docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -30,104 +30,42 @@ The Next.js Starter Kit includes the following
-page), components, which you can think of as multiplayer implementations
-(cursors, badges, etc), pages, which render document level experiences and both
-server side and client side methods which contain the logic to create and modify
-documents.
-
-<br />
-<details>
-  <summary>More info about the structure</summary>
-  <br />
-  <dl>
-    <dt>`/components/`</dt>
-    <dd>Components used in the app.</dd>
-    <dt>`/data/`</dt>
-    <dd>A demo database.</dd>
-    <dt>`/layouts/`</dt>
-    <dd>Page layout components.</dd>
-    <dt>`/lib/client/`</dt>
-    <dd>Client functions used for accessing documents, database, SWR hooks.</dd>
-    <dt>`/lib/server/`</dt>
-    <dd>
-      Server functions used for accessing documents, database, liveblocks.
-    </dd>
-    <dt>`/pages/`</dt>
-    <dd>Next.js pages.</dd>
-    <dt>`/primitives/`</dt>
-    <dd>Generic reusable components.</dd>
-    <dt>`/types/`</dt>
-    <dd>All reusable TypeScript types.</dd>
-    <dt>`/liveblocks.config.ts`</dt>
-    <dd>Liveblocks config file.</dd>
-    <dt>`/liveblocks.server.config.ts`</dt>
-    <dd>Server-side Liveblocks config file.</dd>
-  </dl>
-</details>
+### create-liveblocks-app
 
-### Liveblocks configuration
+You can get started by running the following command:
 
-If you already have a Liveblocks Development account, you may skip to step 4.
+```shell
+npx create-liveblocks-app@latest --next
+```
 
-1. Create an account at [liveblocks.io](https://liveblocks.io])
-2. Navigate to the [dashboard](https://liveblocks.io/dashboard) and click "Add
-   project..."
-3. Create a new project with "Development" as the environment.
-4. Go to "API keys" on the side menu, reveal your secret key, and copy it
+This will run an installer that allows you to download, configure, and deploy, your project:
 
 <Figure>
   <video autoPlay loop muted playsInline>
     <source
-      src="/images/docs/next-js-starter-kit/create-api-key.mp4"
+      src="/images/docs/next-js-starter-kit/create-liveblocks-app.mp4"
       type="video/mp4"
     />
   </video>
 </Figure>
 
-5. From the command line, get the starter kit by running Create Liveblocks App. Follow through the instructions, selecting your project name, chosen authentication method, and install the app:
-
-```shell
-npx create-liveblocks-app --next
-```
-
-6. Enter the `.env.local` file, which is located alongside `package.json` at the
-   root level in your project. This file will contain a randomly generated `NEXTAUTH_SECRET` that will be used for authentication, along with a number of empty variables for you to fill in.
-7. Place your Liveblocks secret key into the file as `LIVEBLOCKS_SECRET_KEY`.
-
-Your `.env.local` file should contain a line similar to this:
-
-```dotenv file=".env.local"
-LIVEBLOCKS_SECRET_KEY =
-  sk_dev_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX;
-```
+Give your project a name, and select the [authentication](#authentication) method you’d like to use in your app.
```

Comment:
Suggesting replacing lines 52-66 with this block (you cant make a suggestion over deleted lines that is why I am mentioning this here).    I'd also consider adding a video for Vercel deployment but that is your call @CTNicholas. attaching one to main PR if you think it makes sense lmk and i'll add it.
```suggestion
Give your project a name, and select the [authentication](#authentication) method you’d like to use in your app.
If you are trying to generate a proof of concept quickly, opting to use demo authentication may be a great option- and you can still add other authentication providers later on.

#### Deploy with Vercel Integration
If you would like to set up CICD with your application, we've made that process straightforward for you as well.  When prompted, you can select "deploy to Vercel" to enable building and deploying the starter kit. The Vercel Integration will open in a new browser window for you to complete the process of adding your repository. If by clicking "Create" you recieve "An unexpected internal error occurred" you should validate within your code hosting platofrom of choice (Github, GitLab, Bitbucket) that the Vercel integration has permission to access your repository.  In Github, permissions can be found can be found under Settings > Integrations > Vercel. 

#### Connect to Liveblocks and retrieve your secret key
 If you prefer to work locally, you can tell the installer you would not like to "deploy to Vercel."  After declining the deployment option, the installer will prompt adding your Liveblocks Key to the application automatically.  If you forwent deployment or indicated that you would like to get your Liveblocks secret key automatically, the Liveblocks integration page will open in a new browser tab. 

Once the integration page appears, you can sign up/sign in to Liveblocks, create a new project, and import your API key into the installer.

<Figure>
  <video autoPlay loop muted playsInline>
    <source
      src="/images/docs/next-js-starter-kit/create-liveblocks-project.mp4"
      type="video/mp4"
    />
  </video>
</Figure>

After finishing up, check the installer and follow the commands recommended to get you started.
```

Reply:
CICD should be CI/CD